External/Plugins/ASCompletion/Controls/AddClosingBracesRulesEditorForm.cs

```diff
@@ -0,0 +1,804 @@
+            btnAddRule.Name = "btnAddRule";
+            btnAddRule.Size = new Size(30, 30);
+            btnAddRule.TabIndex = 9;
+            btnAddRule.Text = "";
+            btnAddRule.UseVisualStyleBackColor = true;
+            btnAddRule.Click += BtnAddRule_Click;
+            // 
+            // btnOk
+            // 
+            btnOk.Anchor = AnchorStyles.Bottom | AnchorStyles.Right;
+            btnOk.DialogResult = DialogResult.OK;
+            btnOk.Location = new Point(724, 311);
+            btnOk.Name = "btnOk";
+            btnOk.Size = new Size(120, 30);
+            btnOk.TabIndex = 10;
+            btnOk.Text = "OK";
+            btnOk.UseVisualStyleBackColor = true;
+            btnOk.Click += BtnOk_Click;
+            // 
+            // btnCancel
+            // 
+            btnCancel.Anchor = AnchorStyles.Bottom | AnchorStyles.Right;
+            btnCancel.DialogResult = DialogResult.Cancel;
+            btnCancel.Location = new Point(850, 311);
+            btnCancel.Name = "btnCancel";
+            btnCancel.Size = new Size(120, 30);
+            btnCancel.TabIndex = 11;
+            btnCancel.Text = "Cancel";
+            btnCancel.UseVisualStyleBackColor = true;
+            btnCancel.Click += BtnCancel_Click;
+            // 
+            // AddClosingBracesRulesEditor
+            // 
+            AcceptButton = btnOk;
+            AutoScaleDimensions = new SizeF(8F, 16F);
+            AutoScaleMode = AutoScaleMode.Font;
+            CancelButton = btnCancel;
+            ClientSize = new Size(982, 353);
+            Controls.Add(btnCancel);
+            Controls.Add(btnOk);
+            Controls.Add(btnAddRule);
+            Controls.Add(btnRemove);
+            Controls.Add(btnAdd);
+            Controls.Add(btnDown);
+            Controls.Add(btnUp);
+            Controls.Add(rulesGridView);
+            Controls.Add(txtCloseChar);
+            Controls.Add(lblCloseChar);
+            Controls.Add(txtOpenChar);
+            Controls.Add(lblOpenChar);
+            Controls.Add(txtName);
+            Controls.Add(lblName);
+            Controls.Add(listBox);
+            FormBorderStyle = FormBorderStyle.SizableToolWindow;
+            MinimumSize = new Size(900, 300);
+            Name = "AddClosingBracesRulesEditor";
+            ShowInTaskbar = false;
+            StartPosition = FormStartPosition.CenterParent;
+            Text = "AddClosingBracesRulesEditor";
+            ((ISupportInitialize) (rulesGridView)).EndInit();
+            ResumeLayout(false);
+        }
+
+        #endregion
+
+        #region Initialization
+
+        private void InitializeGraphics()
+        {
+            colDelete.Image = PluginBase.MainForm.FindImage16("153", false);
+            btnUp.Image = PluginBase.MainForm.FindImage16("74", false);
+            btnDown.Image = PluginBase.MainForm.FindImage16("60", false);
+            btnAddRule.Image = PluginBase.MainForm.FindImage16("33", false); // Seriously no plus icon?
```

Comment:
I guess if you find free to use pictures and they are nice @Meychi will be ok with adding them.